templates/master/01-master-container-runtime/_base/files/crio.yaml

```diff
@@ -130,106 +92,88 @@ contents:
     # List of default capabilities for containers. If it is empty or commented out,
     # only the capabilities defined in the containers json file by the user/kube
     # will be added.
-    # default_capabilities = [
-    # 	"CHOWN",
-    # 	"DAC_OVERRIDE",
-    # 	"FSETID",
-    # 	"FOWNER",
-    # 	"NET_RAW",
-    # 	"SETGID",
-    # 	"SETUID",
-    # 	"SETPCAP",
-    # 	"NET_BIND_SERVICE",
-    # 	"SYS_CHROOT",
-    # 	"KILL",
-    # ]
+    default_capabilities = [
+        "CHOWN", 
+        "DAC_OVERRIDE", 
+        "FSETID", 
+        "FOWNER", 
+        "NET_RAW", 
+        "SETGID", 
+        "SETUID", 
+        "SETPCAP", 
+        "NET_BIND_SERVICE", 
+        "SYS_CHROOT", 
+        "KILL", 
+    ]
 
     # List of default sysctls. If it is empty or commented out, only the sysctls
     # defined in the container json file by the user/kube will be added.
-    # default_sysctls = [
-    # ]
-
+    default_sysctls = [
+    ]
+    
     # List of additional devices. specified as
     # "<device-on-host>:<device-on-container>:<permissions>", for example: "--device=/dev/sdc:/dev/xvdc:rwm".
     #If it is empty or commented out, only the devices
     # defined in the container json file by the user/kube will be added.
-    # additional_devices = [
-    # ]
+    additional_devices = [
+    ]
 
     # Path to OCI hooks directories for automatically executed hooks.
-    # Note: the default is just /usr/share/containers/oci/hooks.d, but /usr is immutable in RHCOS
-    # so we add /etc/containers/oci/hooks.d as well
     hooks_dir = [
-        "/etc/containers/oci/hooks.d",
+        "/etc/containers/oci/hooks.d", 
```

Comment:
injecting trailing whitespace, what!? ;)  Obviously a nit, but maybe worth fixing in whatever source this was pulled from.